surfactant/cmd/generate.py

```diff
@@ -186,7 +231,7 @@ def sbom(
     if not input_sbom:
         new_sbom = SBOM()
     else:
-        new_sbom = SBOM.from_json(input_sbom.read())
+        new_sbom = input_reader.read_sbom(input_sbom)
```

Comment:
Can you make this same change for the merge command in https://github.com/LLNL/Surfactant/blob/CYT-470-add-hook-for-reading-sbom-files/surfactant/cmd/merge.py#L33?

Reply:
I have finished making those change to the merge command. For now, should we assume that all the input SBOMs being merged have the same format?

Reply:
Yes, for now let’s assume they all have the same format, and update docs or argument descriptions to mention that.

Reply:
I have updated the relevant READMEs